### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception
    id: _a_d_l_exception
    parent: com.microsoft.azure.datalake.store
    children:
      - com.microsoft.azure.datalake.store._a_d_l_exception.ADLException(String)
      - 'com.microsoft.azure.datalake.store._a_d_l_exception.ADLException(String,Throwable)'
      - com.microsoft.azure.datalake.store._a_d_l_exception.httpResponseCode
      - com.microsoft.azure.datalake.store._a_d_l_exception.httpResponseMessage
      - com.microsoft.azure.datalake.store._a_d_l_exception.lastCallLatency
      - com.microsoft.azure.datalake.store._a_d_l_exception.numRetries
      - com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionJavaClassName
      - com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionMessage
      - com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionName
      - com.microsoft.azure.datalake.store._a_d_l_exception.requestId
      - com.microsoft.azure.datalake.store._a_d_l_exception.responseContentLength
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: ADLException
    nameWithType: ADLException
    fullName: com.microsoft.azure.datalake.store.ADLException
    type: Class
    source:
      remote: &ref_0
        path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
        branch: master
        repo: 'https://github.com/azure/azure-data-lake-store-java'
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 16
    package: com.microsoft.azure.datalake.store
    summary: <p>例外の種類が Azure Data Lake SDK メソッドによって返されます。 派生した<xref uid="" data-throw-if-not-resolved="false">IOException</xref>です。 成功に関する情報を含む追加のフィールドの数またはサーバーの呼び出しの失敗を表します。 </p>
    syntax:
      content: public class ADLException
    inheritance:
      - java.lang.Object
      - IOException
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.ADLException(String)
    id: ADLException(String)
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: ADLException(String message)
    nameWithType: ADLException.ADLException(String message)
    fullName: com.microsoft.azure.datalake.store.ADLException.ADLException(String message)
    overload: com.microsoft.azure.datalake.store._a_d_l_exception.ADLException*
    type: Constructor
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 67
    package: com.microsoft.azure.datalake.store
    syntax:
      content: public ADLException(String message)
      parameters:
        - id: message
          type: '26831127'
  - uid: 'com.microsoft.azure.datalake.store._a_d_l_exception.ADLException(String,Throwable)'
    id: 'ADLException(String,Throwable)'
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: 'ADLException(String message, Throwable initCause)'
    nameWithType: 'ADLException.ADLException(String message, Throwable initCause)'
    fullName: 'com.microsoft.azure.datalake.store.ADLException.ADLException(String message, Throwable initCause)'
    overload: com.microsoft.azure.datalake.store._a_d_l_exception.ADLException*
    type: Constructor
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 71
    package: com.microsoft.azure.datalake.store
    syntax:
      content: 'public ADLException(String message, Throwable initCause)'
      parameters:
        - id: message
          type: '26831127'
        - id: initCause
          type: a04fafd7
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.httpResponseCode
    id: httpResponseCode
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: httpResponseCode
    nameWithType: ADLException.httpResponseCode
    fullName: int com.microsoft.azure.datalake.store.ADLException.httpResponseCode
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 21
    package: com.microsoft.azure.datalake.store
    summary: <p>サーバーによって返される HTTP 応答コード </p>
    syntax:
      content: public int httpResponseCode
      return:
        type: f75371fa
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.httpResponseMessage
    id: httpResponseMessage
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: httpResponseMessage
    nameWithType: ADLException.httpResponseMessage
    fullName: String com.microsoft.azure.datalake.store.ADLException.httpResponseMessage
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 26
    package: com.microsoft.azure.datalake.store
    summary: <p>HTTP 応答メッセージ </p>
    syntax:
      content: public String httpResponseMessage
      return:
        type: '26831127'
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.lastCallLatency
    id: lastCallLatency
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: lastCallLatency
    nameWithType: ADLException.lastCallLatency
    fullName: long com.microsoft.azure.datalake.store.ADLException.lastCallLatency
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 42
    package: com.microsoft.azure.datalake.store
    summary: <p>呼び出しの待機時間。 再試行を行った場合この最後の再試行の待機時間を格納します。 </p>
    syntax:
      content: public long lastCallLatency= 0
      return:
        type: 0364520f
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.numRetries
    id: numRetries
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: numRetries
    nameWithType: ADLException.numRetries
    fullName: int com.microsoft.azure.datalake.store.ADLException.numRetries
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 36
    package: com.microsoft.azure.datalake.store
    summary: <p>呼び出しが失敗したまでの試行回数 </p>
    syntax:
      content: public int numRetries
      return:
        type: f75371fa
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionJavaClassName
    id: remoteExceptionJavaClassName
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: remoteExceptionJavaClassName
    nameWithType: ADLException.remoteExceptionJavaClassName
    fullName: String com.microsoft.azure.datalake.store.ADLException.remoteExceptionJavaClassName
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 65
    package: com.microsoft.azure.datalake.store
    summary: <p>リモートの例外の java クラス名がサーバーは HTTP エラー メッセージで返されます。 </p>
    syntax:
      content: public String remoteExceptionJavaClassName= null
      return:
        type: '26831127'
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionMessage
    id: remoteExceptionMessage
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: remoteExceptionMessage
    nameWithType: ADLException.remoteExceptionMessage
    fullName: String com.microsoft.azure.datalake.store.ADLException.remoteExceptionMessage
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 60
    package: com.microsoft.azure.datalake.store
    summary: <p>HTTP エラー メッセージ内のサーバーによって返されるリモート例外メッセージ。 </p>
    syntax:
      content: public String remoteExceptionMessage= null
      return:
        type: '26831127'
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.remoteExceptionName
    id: remoteExceptionName
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: remoteExceptionName
    nameWithType: ADLException.remoteExceptionName
    fullName: String com.microsoft.azure.datalake.store.ADLException.remoteExceptionName
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 55
    package: com.microsoft.azure.datalake.store
    summary: <p>HTTP エラー メッセージ内のサーバーによって返されるリモート例外名。 </p>
    syntax:
      content: public String remoteExceptionName= null
      return:
        type: '26831127'
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.requestId
    id: requestId
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: requestId
    nameWithType: ADLException.requestId
    fullName: String com.microsoft.azure.datalake.store.ADLException.requestId
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 31
    package: com.microsoft.azure.datalake.store
    summary: <p>サーバー要求 ID </p>
    syntax:
      content: public String requestId= null
      return:
        type: '26831127'
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.responseContentLength
    id: responseContentLength
    parent: com.microsoft.azure.datalake.store._a_d_l_exception
    href: com.microsoft.azure.datalake.store._a_d_l_exception.yml
    langs:
      - java
    name: responseContentLength
    nameWithType: ADLException.responseContentLength
    fullName: long com.microsoft.azure.datalake.store.ADLException.responseContentLength
    type: Field
    source:
      remote: *ref_0
      path: src/main/java/com/microsoft/azure/datalake/store/ADLException.java
      startLine: 50
    package: com.microsoft.azure.datalake.store
    summary: <p>応答には、1 つが含まれている場合、応答のコンテンツの長さ。 これは、応答 conatined 場合は 0 を指定できます本文なしは、本文がチャンクを使用して送信されたかどうか、または転送エンコードします。 </p>
    syntax:
      content: public long responseContentLength= 0
      return:
        type: 0364520f
references:
  - uid: f75371fa
    spec.java:
      - name: int
        fullName: int
  - uid: '26831127'
    spec.java:
      - name: String
        fullName: String
  - uid: 0364520f
    spec.java:
      - name: long
        fullName: long
  - uid: com.microsoft.azure.datalake.store._a_d_l_exception.ADLException*
    name: ADLException
    nameWithType: ADLException.ADLException
    fullName: com.microsoft.azure.datalake.store.ADLException.ADLException
    package: com.microsoft.azure.datalake.store
  - uid: a04fafd7
    spec.java:
      - name: Throwable
        fullName: Throwable